docker-image/Dockerfile.amd64

```diff
@@ -19,13 +19,17 @@
 FROM calico/bpftool:v5.0-amd64 as bpftool
 
 FROM debian:9.8-slim
-LABEL maintainer "Shaun Crampton <shaun@tigera.io>"
+LABEL maintainer="Shaun Crampton <shaun@tigera.io>"
 
-# Install a backported version of iptables to ensure we have version 1.6.2
-# Similarly for iproute2, we want a more recent version for BPF support.
-RUN printf "deb http://deb.debian.org/debian stretch-backports main\n" > /etc/apt/sources.list.d/backports.list \
-    && apt-get update \
-    && apt-get -t stretch-backports install -y iptables iproute2
+RUN echo 'APT::Default-Release "stable";' > /etc/apt/apt.conf.d/99defaultrelease
+RUN echo 'deb     http://ftp.de.debian.org/debian/    buster main contrib non-free' > /etc/apt/sources.list.d/buster.list
+RUN apt-get update && apt-get install -y -t buster \
+    ipset \
+    iptables \
+    iproute2
+
+RUN update-alternatives --set iptables /usr/sbin/iptables-legacy
+RUN update-alternatives --set ip6tables /usr/sbin/ip6tables-legacy
```

Comment:
Just trying to keep the defaults the same as they've always been in case we run iptables commands elsewhere (in FV that haven't been upgraded for example)